server/index.js

```diff
@@ -1,4 +1,17 @@
 const express = require('express')
 const app = express()
+const helpers = require('../database/index.js');
 
-app.listen(3000, () => console.log('Listening on port 3000'))
\ No newline at end of file
+app.listen(3000, () => console.log('Listening on port 3000'));
+
+app.get('/images', (req, res) => {
+  helpers.getImageInfo((err, data) => {
+    if(err){
+      console.log('Error Getting Image Info from DB:', err);
+      return err;
```

Comment:
What is the purpose of returning the err? wouldn't you want to send this back to the client?

Reply:
Hi Toby, changing it to res.send(err). Will do another PR to my group.